python_wrapper/susi_python/response_parser.py

```diff
@@ -25,26 +25,28 @@ def get_action(jsn):
 
 
 def get_query_response(parsed_dict):
-    ans = parsed_dict['answers'][0]
-
-    data = [Datum(jsn)
-            for jsn in ans['data']]
-
-    metadata = Metadata(ans['metadata'])
-
-    actions = [get_action(jsn)
-               for jsn in ans['actions']]
-    print(actions[::-1])
-    answer = Answer(data, metadata, actions[::-1])
-
     try:
         identity_json = parsed_dict['session']['identity']
         identity = Identity(identity_json)
         session = Session(identity)
     except KeyError:
         session = None
-
-    return QueryResponse(answer,parsed_dict, session)
+    if parsed_dict['answers']:
```

Comment:
Check if the len of parse_dict[answers] is not zero
and update accordingly

Reply:
This returns false if it's empty, it works either way. Should I change it?